static/js/src/advantage/distributor/components/DistributorBuyButton/DistributorBuyButton.tsx

```diff
@@ -64,6 +64,7 @@ const DistributorBuyButton = () => {
           );
           location.href = "/account/checkout";
         }}
+        disabled={products?.length === 0}
```

Comment:
Anyway it will show errors in the checkout page if there's no product. I just added the code blow on onclick though. Would it be better?
```
if (products?.length === 0) {
    return;
}
```

Reply:
Removing this change for now.